stanford-earth/SE3_BLT
# EARTH-000: Changed paragraph modules to be consistent.

# READY FOR REVIEW

# Summary
- Enabled `stanford_paragraph_link_banner` after name change
- Config Export

# Needed By (Date)
- August-ish

# Steps to Test

1. Checkout this branch and branch of [Stanford Paragraph Types](https://github.com/SU-SWS/stanford_paragraph_types/pull/62)
2. config import
3. ensure the module got enabled and the link banner paragraph type exists.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_paragraph_types/pull/62

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
